channeld/channel.c

```diff
@@ -971,6 +996,14 @@ static void send_commit(struct peer *peer)
 		return;
 	}
 
+	/* If we haven't received a packet for > 30 seconds, delay. */
+	if (!peer_recently_active(peer)) {
+		/* Mark this as done and try again. */
+		peer->commit_timer = NULL;
+		start_commit_timer(peer);
```

Comment:
This will send PING every 10ms until we receive PONG. It violates BOLT#1.

>A node sending a ping message:
>- SHOULD NOT send ping messages more often than once every 30 seconds.

Reply:
Ah... I didn't notice that ``peer->expecting_pong`` is checked before sending PING.